sycl/doc/extensions/experimental/sycl_ext_oneapi_kernel_properties.asciidoc

```diff
@@ -363,6 +364,86 @@ diagnostic; invalid combinations that can only be detected at run-time should
 result in an implementation throwing an `exception` with the `errc::invalid`
 error code.
 
+=== Using Properties with Lambda Expressions
+
+When a SYCL kernel is defined via a lambda expression, there is no way to
+define a `get` member function and subsequently no way to embed kernel
+properties. Instead, developers must wrap the lambda expression in an object.
+
+To simplify this usage pattern, this extension defines a `kernel_function`
+that encapsulates a kernel function (which may be a lambda expression) and a
+property list.
+
+NOTE: Developers are free to extend `kernel_function` or define their own
+wrapper classes (e.g., to attach commonly used property lists).
```

Comment:
What does "extend" mean here?  Do you mean "create a class that derives from `kernel_function`?  If so, maybe say that more explicitly?  Also, shouldn't this use the new "[Notes: .... -- end note]" syntax?

Reply:
Right, that's what I meant.  Fixed in 174354294c3fda6aee013c5bddc3da7925473096.

I used the old `NOTE` syntax because this is an old extension, and there are quite a few old-style `NOTE` environments in it already. Do you still think we should use the new note syntax?  Or should I use a second PR to update the format of the whole extension, once we've settled on the wording?

Reply:
> Right, that's what I meant. Fixed in [1743542](https://github.com/intel/llvm/commit/174354294c3fda6aee013c5bddc3da7925473096).

Thanks, this looks good.

> I used the old `NOTE` syntax because this is an old extension

OK, that makes sense.  TBH, I was confused by the syntax "NOTE:" (as opposed to "[NOTE]").  I didn't realize that Asciidoc would format that as a callout.  I do not insist on updating the note syntax, but of course you can if you want.